src/App.tsx

```diff
@@ -1,24 +1,214 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { FilterType } from './types/FilterType';
+import { addTodo, getTodos, removeTodo } from './api/todos';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Notifications } from './components/Notifications/Notifications';
+import { ErrorType } from './types/ErrorType';
+import { NewTodo } from './types/NewTodo';
+import { USER_ID } from './components/USER_ID';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState<ErrorType>(ErrorType.NONE);
```

Comment:
it's better to give variable more meaningful naming
```suggestion
  const [error, setError] = useState<ErrorType>(ErrorType.NONE);
```